proposed/2023/Add-dotnet-search.md

```diff
@@ -0,0 +1,100 @@
+# ***Add `nuget list` Functionality to DotNet***
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- [Nigusu](https://github.com/Nigusu-Allehu) <!-- GitHub username link -->
+- [GitHub Issue](https://github.com/NuGet/Home/issues/6060) <!-- GitHub Issue link -->
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+The specification seeks to add `nuget list` functionality to dotnet. The functionality will be added to dotnet as a search command: `dotnet package search`.  
+## Motivation 
```

Comment:
nitpick: empty line between headings and previous content. This happens multiple places throughout the file.

If you edit the markdown in VSCode, if you install the "MarkdownLint" extension, I believe it'll tell you hints like this.

Reply:
Thank you, the extension helped.